### DISCUSSION TITLE
site admins

### DISCUSSION THREAD

**Discussion by rnadupal** [community/community#64801] [Enterprise] [answered]
### Select Topic Area

Question

### Body

I am using GHE and need your help with account management. My instance has been configured to use SAML-based authentication and I didn't add anyone else as site admins during the initial setup. Is there a way to promote a SAML based user as site admin for the entire org using "ghe-user-promote" command? I have tried it but getting the below user, any pointers are appreciated!

=====
Promote users by using your SAML identity provider or disable SAML-based administrator demotion/promotion within the management console to use this command.
=====

**Comment by nirav-gajera** [comment, marked as answer]
Unfortunately, you cannot use the `ghe-user-promote` command to promote a SAML-based user as a site admin. This is because the `ghe-user-promote` command is only meant to be used for users who are authenticated with GitHub's own OAuth2 authentication system.

If you want to promote a SAML-based user as a site admin, you will need to use the GitHub Enterprise Server management console. To do this, follow these steps:

1. Go to the GitHub Enterprise Server management console.
2. Click on the "Users" tab.
3. Find the user that you want to promote and click on their name.
4. Click on the "Administrative" tab.
5. Check the box next to "Site admin".
6. Click on the "Save" button.

Once you have done this, the user will be promoted as a site admin.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @rnadupal ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
